### ISSUE TITLE
Move ganga lb-dev lesson

### ISSUE THREAD

**Issue by alexpearce** [lhcb/first-analysis-steps#200]
The lesson [“Using Ganga with local projects”](https://lhcb.github.io/second-analysis-steps/01-ganga-with-cmake.html) should move to first-steps as we want to teach it then.

**Comment by alexpearce** [comment]
This will need a call-out added saying why we're using `GaudiExec` here, but `DaVinci` in the previous lessons. Should say that `GaudiExec` will become the default.

**Comment by mesmith75** [comment]
@alexpearce Yes that is a good idea.I think it would be good to change the previous lessons to GaudiExec as well.

**Comment by alexpearce** [comment]
We've discussed that. Right now the thing putting us off is having to create an (effectively empty) `lb-dev` project. We'll be teaching this part before `lb-dev`, so it might be confusing.

**Comment by mesmith75** [comment]
You don't need to.
```python
j = Job()
myApp = prepareGaudiExec('DaVinci','v41r3')
myApp.optsFile = ['some/path/options.py']
j.application = myApp
```
The prepareGaudiExec function builds the standard released version for you.

**Comment by alexpearce** [comment]
Ah! Perfect. Yes, we should move to that then.

Does `prepareGaudiExec` create a temporary directory somewhere? Or will it be created wherever you ran `ganga` from?

**Comment by mesmith75** [comment]
It makes it in your cmtuser area.

**Comment by cmarinbe** [comment]
With #211 all the `ganga` lessons will be using `GaudiExec`, so we can move the lesson without loosing consistency.
